File: 0088-merge-sorted-array/0088-merge-sorted-array.py
class Solution(object):
    def merge(self, nums1, m, nums2, n):
        """
        :type nums1: List[int]
        :type m: int
        :type nums2: List[int]
        :type n: int
        :rtype: None Do not return anything, modify nums1 in-place instead.
        """
        nums3 = []
        left = 0
        right = 0
        index = 0

        while left < m and right < n:
            if nums1[left] < nums2[right]:
                nums3.append(nums1[left])
                left += 1
            else:
                nums3.append(nums2[right])
                right += 1

        while left < m:
            nums3.append(nums1[left])
            left += 1
        while right < n:
            nums3.append(nums2[right])
            right += 1

        for i in range(m+n):
            nums1[i] = nums3[i]
    
        

File: 0088-merge-sorted-array/0088-merge-sorted-array.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        
        int arr [] = new int [m+n];
        int k =0;
        int i = 0, j=0;
        while(i<m){
            arr[k++] = nums1[i++];
        }
        while(j<n){
            arr[k++] = nums2[j++];
        }
        Arrays.sort(arr);
        for(i = 0; i< m+n;i++){
            nums1[i] = arr[i];
        }
        
    }
}